lib/themes/catho/tokens.ts

```diff
@@ -0,0 +1,217 @@
+            rows: 12,
+          },
+          tablet: {
+            size: 36,
+            rows: 12,
+          },
+        },
+        '2': {
+          mobile: {
+            size: 28,
+            rows: 10,
+          },
+          tablet: {
+            size: 28,
+            rows: 10,
+          },
+        },
+        '3': {
+          mobile: {
+            size: 24,
+            rows: 9,
+          },
+          tablet: {
+            size: 24,
+            rows: 9,
+          },
+        },
+        '4': {
+          mobile: {
+            size: 20,
+            rows: 7,
+          },
+          tablet: {
+            size: 20,
+            rows: 7,
+          },
+        },
+      },
+    },
+    text: {
+      xsmall: {
+        mobile: {
+          size: 12,
+          rows: 4,
+        },
+        tablet: {
+          size: 12,
+          rows: 4,
+        },
+      },
+      small: {
+        mobile: {
+          size: 14,
+          rows: 4,
+        },
+        tablet: {
+          size: 14,
+          rows: 4,
+        },
+      },
+      standard: {
+        mobile: {
+          size: 16,
+          rows: 6,
+        },
+        tablet: {
+          size: 16,
+          rows: 6,
+        },
+      },
+      large: {
+        mobile: {
+          size: 18,
+          rows: 6,
+        },
+        tablet: {
+          size: 18,
+          rows: 6,
+        },
+      },
+    },
+  },
+  breakpoint: {
+    mobile: 0,
+    tablet: 600,
+    desktop: 1024,
+  },
+  contentWidth: {
+    medium: 984,
+    large: 1180,
+  },
+  grid: 4,
+  touchableSize: 11,
+  space: {
+    gutter: 6,
+    xxsmall: 1,
+    xsmall: 2,
+    small: 3,
+    medium: 4,
+    large: 6,
+    xlarge: 12,
+    xxlarge: 20,
+  },
+  transforms: {
+    touchable: 'scale(0.97)',
+  },
+  transitions: {
+    fast: 'transform .125s ease, opacity .125s ease',
+    touchable: 'transform 0.2s cubic-bezier(0.02, 1.505, 0.745, 1.235)',
+  },
+  border: {
+    radius: {
+      standard: '4px',
+    },
+    width: {
+      standard: 1,
+      large: 2,
+    },
+    color: {
+      standard: '#999999',
+      standardInverted: white,
+      focus,
+      critical,
+      info,
+      promote,
+      positive,
+      caution,
+      formHover: formAccent,
+      formAccent,
+    },
+  },
+  shadows: {
+    small:
+      '0 2px 4px 0px rgba(28,28,28,.1), 0 2px 2px -2px rgba(28,28,28,.1), 0 4px 4px -4px rgba(28,28,28,.2)',
+    medium:
+      '0 2px 4px 0px rgba(28,28,28,.1), 0 8px 8px -4px rgba(28,28,28,.1), 0 12px 12px -8px rgba(28,28,28,.2)',
+    large:
+      '0 2px 4px 0px rgba(28,28,28,.1), 0 12px 12px -4px rgba(28,28,28,.1), 0 20px 20px -12px rgba(28,28,28,.2)',
+  },
+  color: {
+    foreground: {
+      link,
+      linkHover: link,
+      linkVisited,
+      neutral,
+      neutralInverted: white,
+      formAccent,
+      critical,
+      info,
+      promote,
+      caution,
+      positive,
+      secondary,
+      secondaryInverted: 'hsla(0, 0%, 100%, 0.65)',
```

Comment:
Looks like IE11 doesn't support `hsla`: https://caniuse.com/#feat=mdn-css_types_color_hsl_function_accepts_alpha

Reply:
Looks like we use a similar technique across all themes. Might be something we should address holistically.

Reply:
Hang on looks like we might be misreading, `hsl` does not accept an `alpha` value in IE11, but `hsla` does https://caniuse.com/#feat=mdn-css_types_color_alpha

Reply:
Ah yep, gotcha. Carry on.